vite-project-kabinet/src/Content/PaginatedList.jsx

```diff
@@ -0,0 +1,53 @@
+import "./paginated.style.css"
+import { Card } from "./Card.jsx"
+import { useState } from "react";
+import arrowRight from "./assets/sipka_vpravo.svg"
+import arrowLeft from "./assets/sipka_vlevo.svg"
+
+export const PaginatedList = ({items, itemsPerPage}) => {
+    const [currentPage, setCurrentPage] = useState(1);
+
+    const totalPages = Math.ceil(items.length / itemsPerPage);
+
+    const handlePageChange = (newPage) => {
+        if (newPage >= 1 && newPage <= totalPages) {
+            setCurrentPage(newPage);
+        }
+    };
+
+    return (
+        <>
+            <section className="card__container">
+                {/* Zobrazení seznamu položek - tady namapuju komponentu <Card/> */}
+                {items
+                    .slice((currentPage - 1) * itemsPerPage, currentPage * itemsPerPage)
+                    .map((item) => (
+                    <Card key={item.id} genre={item.genre} title={item.title} author={item.author} content={item.content} image={item.image} />
+                ))}
+            </section>
+
+            {/* Ovládací prvky*/}
+            <div className="pages">
+                <button 
+                    onClick={() => handlePageChange(currentPage - 1)}
+                    disabled={currentPage === 1}
+                >
+                    <img src={arrowLeft} alt="šipka vlevo"/>
+                </button>
+
+                <ul className="pages__numbers">
+                    <li>
+                        <button>{currentPage}</button>
+                    </li>
+                </ul>
+
+                <button
+                    onClick={() => handlePageChange(currentPage + 1)}
+                    disabled={currentPage === totalPages}
+                >
+                    <img src={arrowRight} alt="šipka vpravo"/>
+                </button>
+            </div>
+        </>
+    )
+  }
```

Comment:
Ešte ma napadlo, nemám z tohoto kusu kódu vytvoriť komponentu?

Reply:
Dobrá úvaha. 
Z komentáře není jasné, co přesně myslíš. Ale jestli to samotné stránkování, tedy to pod komentářem "Ovladaci prvky" tak to určitě bude lepší jako samostatná komponenta.